cmds/core/printf/utils.go

```diff
@@ -0,0 +1,211 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+func interpret(
```

Comment:
I'm a bit surprised to see you had to write your own interpreter, is the one in fmt not usable for some reason?

Reply:
the fmt in golang is semantically different from every implementation that I compared with (coreutils, bash, zsh, sh) 

I'm pretty sure this will be inevitable, especially if we need to implement format code %q